x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java

```diff
@@ -60,7 +60,9 @@
 public class EnrichPlugin extends Plugin implements ActionPlugin, IngestPlugin {
 
     static final Setting<Integer> ENRICH_FETCH_SIZE_SETTING =
-        Setting.intSetting("index.xpack.enrich.fetch_size", 10000, 1, 1000000, Setting.Property.NodeScope);
+        Setting.intSetting("xpack.enrich.fetch_size", 10000, 1, 1000000, Setting.Property.NodeScope);
```

Comment:
outside the scope of this review .. .but 'fetch_size' should probably be a dynamic setting

Reply:
are we still calling these `xpack.*`? I thought our goal was to go away from using xpack